Makefile.buildtests

```diff
@@ -38,18 +38,18 @@ buildtest:
 	@ \
 	BUILDTESTOK=true; \
 	APP_RETRY=0; \
-	rm -rf "$$BINDIRBASE"; \
```

Comment:
Yep.
(```$BINDIRBASE``` would be "tests/unittests/bin2, while "make clean" only cleans "tests/unittests/bin/<board>", but for the build that makes no difference.)